lib/dpl/providers/codedeploy.rb

```diff
@@ -84,6 +84,7 @@ def create_deployment
       def wait_until_deployed(id)
         print :waiting_for_deploy
         status = poll(id) until %w(Succeeded Failed Stopped).include?(status)
+        error "Waiting terminated with status: #{status}" if status != 'Succeeded'
```

Comment:
The message seems rather awkward. How about simply changing how we exit this method?

```ruby
    case status
    when 'Succeeded'
      info :finished_deploy, status # or set up a new message template
    else
      error :finished_deploy, status
    end
```

Reply:
Yikes, I don't know how I missed this, gotta get my GH notifications under control I guess. ;)

I'm totally fine with that, it's a nice simplification.